src/components/svg/wrap/svg-keyboard-wrap.component.scss

```diff
@@ -10,12 +10,34 @@ svg-keyboard {
     max-width: 1400px;
     position: absolute;
     left: 0;
-    transform: translateX(-100%);
+    transform: translateX(-101%);
```

Comment:
It was changed, because there where some traces of keyboard that supposed to be hidden.